documentation/src/main/asciidoc/topics/con_restarting_clusters.adoc

```diff
@@ -1,10 +1,14 @@
 [id='cluster-restarts_{context}']
 = {brandname} cluster restarts
 
-When you bring {brandname} clusters back online after shutting them down, you should wait for the cluster to be available before adding or removing nodes or modifying cluster state.
+When you bring {brandname} clusters back online after using the `shutdown cluster` command, all nodes and caches in the cluster will be unavailable until all nodes rejoin.
+To prevent inconsistencies or data loss, {brandname} restricts access to the data stored in the cluster and modifications of the cluster state until the cluster is fully operational again.
+Additionally, {brandname} disables cluster rebalancing and prevents file stores purging on startup.
+
+{brandname} logs messages for each node during the cluster recovery process, indicating which nodes are available and which are still missing. +
```

Comment:
The messages are repeated on the coordinator only (if nodes are started), but all other nodes will only show that message at startup. @jabolina correct, right?

Reply:
Thank you. Now it's updated